tools/zapdir/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/Microsoft/hcsshim/internal/wclayer"
+	"github.com/urfave/cli"
+)
+
+const (
+	dirArgName = "dir"
+)
+
+func main() {
+	app := cli.NewApp()
```

Comment:
Given the simplicity of the app can we just use flags ourselves? This seems overkill

Reply:
Keep it there. I have an in-progress proof of concept change which will add some other flags to this

Reply:
I like knowing I can use --help to get help output, and also the consistency of double-dash args, and having the same pattern to add a new arg across our code. I think these are worth it, but will change it if you want me to.